doc/source/how-to/documenting.rst

```diff
@@ -375,9 +383,9 @@ and
 classes are shared across multiple queries. Consequently, these classes are
 documented separately.
 
-
 Build documentation
 -------------------
+
 `Sphinx <https://www.sphinx-doc.org/en/master/>`_ is used to build the documentation.
 You configure the entire build process in the ``conf.py`` file, located in the
 ``source/`` directory in :ref:`The \`\`doc/\`\` directory`.
```

Comment:
I've seen a few comments about how seeing "The" in the heading and a link is odd. I agree but also feel using a code entity in the heading is odd too. For now, I'm OK with leaving this--but perhaps we should revisit the topic someday.